### PR TITLE
Feature/16 sign in screen

### LINKED ISSUE #16: Add Sign in screen design
# REQUIREMENTS
This screen will display after touch the "Iniciar Sesión" button on the main screen of the app. The screen will require the user's email and password to sign in into the app. Also, this screen has two additional buttons:
- The "¿no tienes cuenta? Registrate" button, this button will send the users into the four steps to create his new account.
- The "¿Olvidaste tu clave?" button that redirects the user into a new screen with some steps to create a new password 

# DESIGN
image

# Acceptance Criteria
- [ ] Implementation must match the design provided
- [ ] No functionality is required yet, only design

### PR DESCRIPTION
**Pull request by OwenRamire**
# Description
- This new form is for the users that they were registered before, and they want to use his account 
- In case that the user touches the button "Iniciar sesión" and doesn't have an account, this form has a button called "¿No tienes cuenta? Regístrate" that will send the user into the SignUp forms.

# Screenshots 
- Layout of the screen

Screenshot_1602617340

- Go SignIn screen

GoSignIn

- Go SignUp forms from SignIn screen

GoSignUpOnSignIn

### CODE REVIEW

File: src/navigations/NavigationSignIn.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const SignInStack = createStackNavigator();
+
+const NavigationSignIn = () => {
+  return (
+    <SignInStack.Navigator>
+      <SignInStack.Screen
+        name="SignIn"
+        options={{
+          title: 'Inicio de sesión',
```
Comment (fernandoamz) [inline src/navigations/NavigationSignIn.js]: Add to constants file

File: src/screens/signInScreens/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+const SignIn = ({navigation}) => {
+  const inputType = ['Correo electrónico', 'Clave'];
+
+  const showInputs = () => {
+    const inputs = inputType.map((type) => {
+      if (type === 'Correo electrónico') {
+        return (
+          <EmailPassTxtInput
+            key={type}
+            title="Correo electrónico"
```
Comment (fernandoamz) [inline src/screens/signInScreens/SignIn.js]: Add to constants file

File: src/screens/signInScreens/SignIn.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            password={false}
+          />
+        );
+      } else {
+        return <EmailPassTxtInput key={type} title="Clave" password={true} />;
+      }
```
Comment (fernandoamz) [inline src/screens/signInScreens/SignIn.js]:
```suggestion
      } 
      
      return <EmailPassTxtInput key={type} title="Clave" password={true} />;
```

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by xErikPsx — CHANGES_REQUESTED** [review]
The mentor Fernando left some comments, those could be cool to change.

**Comment by fernandoamz** [comment]
@xErikPsx, Sorry, I didn't see this name of the screen `NavigationSignIn` I think that is better if you choose a better name without the `Navigation` prefix this will be redundant in the next screens.

File: src/screens/signInScreens/SignIn.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  const inputType = [EMAIL, PASSWORD];
+
+  const showInputs = () => {
+    const inputs = inputType.map((type) => {
+      if (type === EMAIL) {
+        return <EmailPassTxtInput key={type} title={EMAIL} password={false} />;
+      }
+      return <EmailPassTxtInput key={type} title={PASSWORD} password={true} />;
+    });
+    return inputs;
```
Comment (fernandoamz) [inline src/screens/signInScreens/SignIn.js]:
```suggestion
    return inputType.map((type) => {
        if (type === EMAIL) {
          return <EmailPassTxtInput key={type} title={EMAIL} password={false} />;
        }
        return <EmailPassTxtInput key={type} title={PASSWORD} password={true} />;
    });
```

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by fernandoamz — APPROVED** [review]

**Review by luisarmandodv — APPROVED** [review]

**Pull request merged**